### PR TITLE
Adamfuller zz patch 1

### PR DESCRIPTION
**Pull request by adamfuller-zz**
Hi Kip, I wound up closing my original PR and creating this new one.

### CODE REVIEW

File: geotiff/geotiff.py [hunk 1]
```diff
@@ -182,7 +182,11 @@ def tif_bBox_wgs_84(self) -> BBox:
     def _get_crs_code(self, geotiff_metadata: dict) -> int:
         temp_crs_code: Optional[int] = None
         if geotiff_metadata["GTModelTypeGeoKey"].value == 1:
-            temp_crs_code = geotiff_metadata["ProjectedCSTypeGeoKey"].value
+            if hasattr(geotiff_metadata["ProjectedCSTypeGeoKey"], 'value'):
+                temp_crs_code = geotiff_metadata["ProjectedCSTypeGeoKey"].value
+            else
```
Comment (KipCrossing) [inline geotiff/geotiff.py:187]: @adamfuller-zz Looks like you left out a `:` on your else statement

**Review by KipCrossing — CHANGES_REQUESTED** [review]

**Comment by KipCrossing** [comment]
You can test your changes by running:

```
python -m pytest
```

You may need to install `pytest`

**Comment by adamfuller-zz** [comment]
God not a great first outing.  Yep I thought I would be clever and make my commit through the github web editor without actually running the code (or using pytest, I'd never used it but I installed it and have run it successfully now yay).  Unfortunately for geotiff, this is the first public repo I have contributed to, your karma is definately +1 now.

**Review by KipCrossing — APPROVED** [review]
Looks good!

**Comment by KipCrossing** [comment]
> God not a great first outing. Yep I thought I would be clever and make my commit through the github web editor without actually running the code (or using pytest, I'd never used it but I installed it and have run it successfully now yay). Unfortunately for geotiff, this is the first public repo I have contributed to, your karma is definately +1 now.

All good. A great way to learn is to make mistakes - especially in a public setting. I welcome all patches as it's impossible for me to know about all the edge cases; such as this one. Thanks for the PR

**Comment by KipCrossing** [comment]
The new version it up the your fix: https://pypi.org/project/geotiff/0.2.3/

**Pull request merged**